core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -120,6 +120,9 @@ class StageData private[spark](
     val numFailedTasks: Int,
 
     val executorRunTime: Long,
+    var submissionTime: Date,
```

Comment:
oh right, I was testing it as "Long" at the beginning, and then changed to "Date" but forgot to change it to be val. I'll change it.